javascriptv3/example_code/medical-imaging/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "example-javascriptv3-medical-imaging",
+  "version": "1.0.0",
+  "author": "Corey Pyle <corepyle@amazon.com",
```

```suggestion
  "author": "Corey Pyle <corepyle@amazon.com>",
```
But, feel free to put your own name here.